### ISSUE TITLE
Searchbox input is missing a label

### ISSUE THREAD

**Issue by bjcfuller** [uriweb/uri-modern#64]
## Issue Type

[] Bug  
[] New Feature  
[X] Enhancement  
    
## Description

Need a label for `<input role="searchbox" ...>` in `brandbar.php`
